app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -1,6 +1,12 @@
-attributes :content_view_id, :content_view_name
-attributes :lifecycle_environment_id, :lifecycle_environment_name
-attributes :uuid
+child :content_view => :content_view do
+  attributes :id, :name
+end
+
+child :lifecycle_environment => :lifecycle_environment do
+  attributes :id, :name
+end
```

Comment:
I'd prefer we'd use the object method everywhere.  Either way, we have to choose consistency with katello APIs or consistency with foreman APIs.  Since we don't have an agreed upon convention yet I think it is best to include both methods for now.